qa/tasks/cephfs/filesystem.py

```diff
@@ -835,8 +835,8 @@ def create(self, **kwargs):
                       sv_name,
                       '--group_name', 'qa',
                     ]
-                    if subvol_options:
-                        cmd.append(subvol_options)
+                    if svopts:
+                        cmd.append(svopts)
```

Comment:
Same -  this commit still replace `subvol_options` by `svopts`.

Reply:
restored usage of `subvol_options`